File: Java/Amstrong.java
import java.util.Scanner;

public class Amstrong
{


    public void Problem(int num)
    {

         int a=num;
         int sum=0;

          while( num >0)
          {

          	int store = num % 10 ;
            sum = sum+ store * store * store;   
             num= num/10;
           }

          
          String check =   a == sum ? "AmstrongNumber" : "NoAmstrongNumber";
          System.out.println(check);


    }





	public static void main(String args[])
	{

           Scanner input = new Scanner(System.in);
           Amstrong value = new Amstrong();
           int num= input.nextInt();
           value.Problem(num);

	}
}

File: Java/javaScript/Amstrong.js
function number(sum)
{
	let a=sum;
     
	 // the  store the number 
	 let store=0;
  while(a > 0)
  {
	  let temp=a%10;
	   store = store + temp* temp* temp
	   a=parseInt(a/10);
	   
  }
  
    let check= sum == store ? "AmstrongNumber" :"Not AmstrongNumber";
	
	return check;
	
	
}
let value=number(154);
console.log(value);